src/app/gigs/gigs-dialog.tsx

```diff
@@ -0,0 +1,704 @@
+'use client';
+
+import {
+  createContract,
+  updateContractStatus,
+  updateMilestoneStatus,
+  updateOfferStatus,
+} from '@/api/gigs';
+import { ChatLayoutMinimal } from '@/app/gigs/chat';
+import {
+  ContractFormContainer,
+  ContractFormFixedPriceData,
+  ContractMilestoneMinimalProps,
+  ContractMilestonesFormData,
+  ContractNotFound,
+  ContractOverview,
+  ContractPendingFreelancer,
+  ContractPendingOffer,
+  ContractRejected,
+} from '@/app/gigs/contract';
+import { useWalletSelector } from '@/components/common';
+import { Spinner } from '@/components/common/spinner';
+import { Button } from '@/components/ui/button';
+import {
+  Dialog,
+  DialogContent,
+  DialogOverlay,
+  DialogTitle,
+} from '@/components/ui/dialog';
+import { useToast } from '@/components/ui/use-toast';
+import { CONTRACT_ID } from '@/constant/constant';
+import { useAuth, useGetContractForOffer } from '@/hooks';
+import { useLocalStorage } from '@/hooks';
+import { toYoctoNEAR } from '@/lib/helper';
+import { cn } from '@/lib/utils';
+import {
+  ContractType,
+  CreateContractType,
+  MilestoneType,
+  OfferType,
+} from '@/types';
+import { Transaction } from '@near-wallet-selector/core';
+import BigNumber from 'bignumber.js';
+import {
+  FileCheck2 as IconContract,
+  MessageCircle as IconMessage,
+} from 'lucide-react';
+import React, { useEffect, useState } from 'react';
+
+export interface GigsDialogProps {
+  isOpen?: boolean;
+  isClient?: boolean;
+  chatParticipantName?: string;
+  currentOfferId?: string;
+  currentOffer?: OfferType;
+  onOpenChange: (open: boolean) => void;
+  handleRejectOffer?: () => void;
+  afterContractSubmitted?: () => void;
+}
+
+export enum ActiveTab {
+  Chat,
+  Contract,
+}
+
+interface TabSelectorProps {
+  activeTab: ActiveTab;
+  setActiveTab: (tab: ActiveTab) => void;
+}
+
+const TabSelector: React.FC<TabSelectorProps> = ({
+  activeTab,
+  setActiveTab,
+}) => {
+  return (
+    <div className='border border-1 border-solid border-gray-100 rounded-full h-11'>
+      <Button
+        variant={`outline`}
+        className={cn(
+          'rounded-full border-none outline-none gap-2 active:ouline-none focus:outline-none hover:bg-sorbet hover:text-white',
+          `${activeTab === ActiveTab.Chat && `bg-sorbet text-white`}`
```

Comment:
`cn` can accept falsy values so you can just do
```suggestion
          activeTab === ActiveTab.Chat && 'bg-sorbet text-white'
```